web/packages/teleport/src/Integrations/Enroll/AwsConsole/IamIntegration/IamIntegration.tsx

```diff
@@ -0,0 +1,434 @@
+    if (!validator.validate()) {
+      return;
+    }
+
+    setScriptUrl(
+      cfg.getAwsRolesAnywhereGenerateUrl(
+        formState.integrationName,
+        formState.trustAnchorName,
+        formState.syncRoleName,
+        formState.syncProfileName
+      )
+    );
+    setPhase(Phase.Two);
+  }
+
+  const manageAction = {
+    content: (
+      <Link href={rolesAnywhere} target="_blank">
+        Manage AWS Roles Anywhere Profiles
+        <NewTab size={18} ml={2} />
+      </Link>
+    ),
+  };
+
+  const testCfg = useMutation({
+    mutationFn: () => {
+      const inputs = parseOutput(sync);
+      // set output for final redirect
+      setOutput({
+        trustAnchorArn: inputs.trustAnchorArn,
+        syncProfileArn: inputs.syncProfileArn,
+        syncRoleArn: inputs.syncRoleArn,
+      });
+      return integrationService.awsRolesAnywherePing(
+        formState.integrationName,
+        inputs.trustAnchorArn,
+        inputs.syncRoleArn,
+        inputs.syncProfileArn
+      );
+    },
+    onError: error => {
+      setAlert(
+        <Alert kind="danger" details={error.message}>
+          Error: {error.name}
+        </Alert>
+      );
+    },
+    onSuccess: (response: AwsRolesAnywherePingResponse) => {
+      if (response.profileCount >= 0) {
+        setAlert(
+          <Alert kind="success" secondaryAction={manageAction}>
+            Success! AWS IAM Roles Anywhere Profiles found
+          </Alert>
+        );
+      } else {
+        setAlert(
+          <Alert
+            kind="warning"
+            secondaryAction={manageAction}
+            details={
+              'Create your first Profile to start accessing AWS from Teleport.'
+            }
+          >
+            Teleport didn&#39;t find any Profiles
+          </Alert>
+        );
+      }
+
+      setPhase(Phase.Three);
+    },
+  });
+
+  const validInput = useMemo(() => {
+    return validTrustAnchorInput(sync)().valid;
+  }, [sync]);
+
+  if (!canEnroll) {
+    return (
+      <FeatureBox>
+        <Alert kind="info" mt={4}>
+          You do not have permission to enroll integrations. Missing role
+          permissions: <code>integrations.create</code>
+        </Alert>
+        <EmptyState />
+      </FeatureBox>
+    );
+  }
+
+  // verify that the parent aws oidc integration exists/is valid
+  if (statsAttempt.status === 'error') {
+    return (
+      <Alerts.Danger details={statsAttempt.error.message}>
+        Error: {statsAttempt.error.name}
+      </Alerts.Danger>
+    );
+  }
+
+  return (
+    <>
+      <Flex mt={3} justifyContent="space-between" alignItems="center">
+        <H2>AWS CLI / Console Access</H2>
+        <InfoGuideButton
+          config={{
+            guide: <Guide resourcesRoute={resourceRoute} />,
+          }}
+        />
+      </Flex>
+      <Box>
+        Compatible with any CLI and AWS SDK-based tootling (includes Terraform,
+        AWS CLI). Teleport uses <b>AWS IAM Roles Anywhere</b> to manage access
```

Comment:
i think that should be a and?
```suggestion
        Compatible with any CLI and AWS SDK-based tooling (includes Terraform
        and AWS CLI). Teleport uses <b>AWS IAM Roles Anywhere</b> to manage access
```